crates/napi/src/next_api/project.rs

```diff
@@ -1006,24 +1006,28 @@ pub fn project_entrypoints_subscribe(
         move || {
             async move {
                 let entrypoints_with_issues_op = get_entrypoints_with_issues_operation(container);
-                let EntrypointsWithIssues {
-                    entrypoints,
-                    issues,
-                    diagnostics,
-                    effects,
-                } = &*entrypoints_with_issues_op
-                    .read_strongly_consistent()
-                    .await?;
+                let (entrypoints_with_issues, issues, diagnostics, effects) =
```

Comment:
not sure if this would cause a problem but both `get_entrypoints_with_issues_operation` and `strongly_consistent_catch_collectables` are collecting issues. 

afaict they are doing it from different VCs, but i think what you should do is just delete `get_entrypoints_with_issues_operation` and instead just call  `project_container_entrypoints_operation` here